### PR TITLE
Add directory for github templates

### PR DESCRIPTION
**Pull request by dwillmer**

### CODE REVIEW

**Comment by dwillmer** [comment]
This is just a trivial PR template - @fastats/core , please add to this if you think i've missed something.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+New Feature
+
+- [ ] Clean code (small functions, proper variable names, PEP8 etc)
+- [ ] Doctests
+- [ ] Unit-tests
+- [ ] Documentation
```
Comment (pawroman) [inline .github/PULL_REQUEST_TEMPLATE.md]: We have no documentation setup yet, do you mean docstrings here?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- [ ] New line item in releases.md
```
Comment (pawroman) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should be `[releases.md](releases.md)` to render as: [releases.md](releases.md)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
+New Feature
+
+- [ ] Clean code (small functions, proper variable names, PEP8 etc)
```
Comment (pawroman) [inline .github/PULL_REQUEST_TEMPLATE.md]: Worth mentioning the style guide https://github.com/fastats/fastats/blob/c7127198686464f5906ffbf04327bf1e565fde4e/CONTRIBUTING.md#code-style

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (dwillmer) [reply .github/PULL_REQUEST_TEMPLATE.md]: No, I mean documentation, I have it mostly ready on another branch

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (dwillmer) [reply .github/PULL_REQUEST_TEMPLATE.md]: nice one, thanks

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+
+New Feature
+
+- [ ] Code conforms to style guide
+- [ ] Doctests
+- [ ] Unit-tests
+- [ ] Documentation
+- [ ] New line item in [releases.md](releases.md)
+- [ ] Info added to `literature` folder
```
Comment (pawroman) [inline .github/PULL_REQUEST_TEMPLATE.md]: Let's not encourage the stupid microsoft contraption of `folder`. It's a `directory`.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (pawroman) [reply .github/PULL_REQUEST_TEMPLATE.md]: Awesome, looking forward to the PR

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (dwillmer) [reply .github/PULL_REQUEST_TEMPLATE.md]: 😄

**Pull request merged**